### DISCUSSION TITLE
Can you support Arclight server？

### DISCUSSION THREAD

**Discussion by 1233Ao** [EssentialsX/Essentials#5093] [Ideas and Feature Suggestions] [answered]
Arclight support plugins and forge mods,it has full support to Bukkit and Spigot plugins. Paper not yet.
You can access its github（https://github.com/IzzelAliz/Arclight）
Arclight is a really great server.It's not the same as Mohist
I just hope Essentials can support Arclight
Thanks

**Comment by 1233Ao** [comment]
its github https://github.com/IzzelAliz/Arclight

**Comment by pop4959** [comment, marked as answer]
From the EssentialsX release notes:

"Other Forge/Bukkit hybrid servers including Cauldron, Thermos, Magma, CatServer - EssentialsX is built for the Bukkit API, which does not properly support mods. For modded servers, use [SpongeForge](https://www.spongepowered.org/) and [Nucleus](https://nucleuspowered.org/), or an alternative mod for [Forge or Fabric](https://www.curseforge.com/minecraft/mc-mods)."

This includes Arclight.

**Comment by 1233Ao** [comment]
okey
